vertx-web/src/main/java/io/vertx/ext/web/impl/Utils.java

```diff
@@ -91,12 +94,33 @@ public static String readFileToString(Vertx vertx, String resource, Charset char
     }
   }
 
-  public static DateFormat createRFC1123DateTimeFormatter() {
-    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.ENGLISH);
-    dtf.setTimeZone(TimeZone.getTimeZone("GMT"));
-    return dtf;
+  public static DateTimeFormatter createRFC1123DateTimeFormatter() {
+//    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.ENGLISH);
+//    dtf.setTimeZone(TimeZone.getTimeZone("GMT"));
+//    return dtf;
+	  return DateTimeFormatter.RFC_1123_DATE_TIME;
+  }
+  
+  private static final ZoneId ZONE_GMT = ZoneId.of("GMT");
+
+  private static String format(final long time,final DateTimeFormatter format) {
+     return format.format(Instant.ofEpochMilli(time).atZone(ZONE_GMT));
   }
 
+  public static String formatRFC1123DateTime(final long time) {
+     return format(time,DateTimeFormatter.RFC_1123_DATE_TIME);
+  }
+  
+  public static long parseREF1123DateTime(final String header) {
+	  try {
+		  return header == null || header.isEmpty() ? -1 :
```

Comment:
Keep the right formatting: indent = 2 spaces

Reply:
ok:)